command/task_delete.go

```diff
@@ -0,0 +1,106 @@
+package command
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/mitchellh/go-wordwrap"
+)
+
+const cmdTaskDeleteName = "task delete"
+
+// TaskDeleteCommand handles the `task delete` command
+type taskDeleteCommand struct {
+	meta
+
+	flags *flag.FlagSet
+}
+
+func newTaskDeleteCommand(m meta) *taskDeleteCommand {
+	flags := m.defaultFlagSet(cmdTaskDeleteName)
+	return &taskDeleteCommand{
+		meta:  m,
+		flags: flags,
+	}
+}
+
+// Name returns the subcommand
+func (c taskDeleteCommand) Name() string {
+	return cmdTaskDeleteName
+}
+
+// Help returns the command's usage, list of flags, and examples
+func (c *taskDeleteCommand) Help() string {
+	helpText := fmt.Sprintf(`
+Usage: consul-terraform-sync task delete [options] <task name>
+
+  Task Delete is used to delete existing tasks. Will not delete a task
```

Comment:
```suggestion
  Task Delete is used to delete an existing task. Will not delete a task
```

Since we're not supporting bulk delete at this time, it might be more clear to just keep "task" consistently singular

Reply:
I've committed your suggestion and will add all the changes for plural to singular!